components/Carousel.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react";
```

Comment:
Can this be incorporated into the 'HorizontalCards.tsx' file as they both are virtually the same component? Looks like your version is better than HorizontalCards, but can you either edit that file or replace it with your code?
Note: User Profile uses HorizontalCards as well

Reply:
Depends, the Carousel specifically is for images where on the profile im assuming it will display different information. however, the carousel can be used for any image preview.